Header header
string[] names
LSPBTrajectorySegmentMsg[] segments
